Kitodo/src/main/java/org/kitodo/config/ConfigCore.java

```diff
@@ -175,4 +180,10 @@ public static Duration getDurationParameter(ParameterCore key, TimeUnit timeUnit
     public static String getKitodoDiagramDirectory() {
         return getParameter(ParameterCore.DIR_DIAGRAMS);
     }
+
+    private static void checkKeyType(ParameterCore key) {
```

Comment:
Please name it something like "checkKeyTypeNotNoll"

Reply:
Done.